api-docs/kotlin/src/test/kotlin/org/bitcoindevkit/Samples.kt

```diff
@@ -86,7 +90,7 @@ fun addressInfoSample() {
     )
 
     fun getLastUnusedAddress(): AddressInfo {
```

Comment:
Looks like a copy/paste error, should be renamed here and on line 96.
```suggestion
    fun getNewAddress(): AddressInfo {
```

Reply:
Yeah thanks for pointing this out I found a few things in there I didn't like so I ended up simplifying the sample a bit. 👍